integrations/electricity/linky.js

```diff
@@ -118,82 +42,64 @@ function disconnect() {
   return {};
 }
 
-async function fetchActivities(frequency, startDate, endDate, logger) {
-  const query = {
-    p_p_col_pos: 1,
-    p_p_lifecycle: 2,
-    p_p_col_count: 2,
-    p_p_state: 'normal',
-    p_p_mode: 'view',
-    p_p_cacheability: 'cacheLevelPage',
-    p_p_col_id: 'column-1',
-    p_p_id: PPID,
-    p_p_resource_id: GRANULARITY[frequency],
-  };
-
-  const payload = {};
-  payload[`_${PPID}_dateDebut`] = startDate;
-  payload[`_${PPID}_dateFin`] = endDate;
-
-  const res = await agent
-    .post('https://espace-client-particuliers.enedis.fr/group/espace-particuliers/suivi-de-consommation')
-    .query(query)
-    .type('form')
-    // Those are required to avoid the client to cache responses
-    .set('Cache-Control', 'no-cache')
-    .set('If-None-Match', '*')
-    .send(payload);
-
-  const responseURL = getResponseURL(res);
-
-  // Check if the response URL is a redirect to the login page
-  if (responseURL && responseURL.includes('Login')) {
-    throw new Error('We\'re supposed to be logged in at this stage');
+async function fetchActivities(usagePointId, frequency, startDate, endDate, logger) {
+  let url;
+  if (frequency === 'hour') {
+    url = '/v3/metering_data/consumption_load_curve';
+  } else if (frequency === 'day') {
+    url = '/v3/metering_data/daily_consumption';
   }
```

Comment:
Maybe something more clear and manageable like this:
```suggestion
  const endpoints = {
    hour: 'consumption_load_curve',
    day: 'daily_consumption',
  };
  const url = `/v3/metering_data/${endpoints[frequency]}`;
```